portfolio-edit/edit-project.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+include './pullPortfolio.php';
+$id = $_GET['id'];
+$projects = pullPortfolio();
```

Comment:
this is quite a slow way of getting 1 project, you can write a new function that gets just 1 project based on the ID from the line above